Redoing the landing page to emphasize the coderetreat format

rachelcarmena opened the pull request:
Marie-Kondo-moment as we commented at the meeting:
* Website title: Coderetreat (instead of Global Day of Coderetreat)
* Navigation: emphasizing GDCR for finding and registering events
* First section: only Coderetreat
* Second section: only Global Day of Coderetreat
* Generic section about Hosting
* Removing the overuse of headings to highlight the text.
* Short version of 10 years of Coderetreat page as an announcement

rradczewski commented:
Thanks for incorporating the changes @rachelcarmena :tada:, I especially like that the header is much more inviting this way!

One thing that I noticed is that we never explain what GDCR stands for and thus it looks a bit cryptic to me on the menu bar. I understand the trouble, "Find a coderetreat" and "Register a coderetreat" together are long and add to the overall size of the menu....

Maybe it'd be even better to remove "Find" and "Register" from the menu altogether? We not gonna open the registration any time soon, so you can't really find nor register an event for 2019 :sweat_smile:

rachelcarmena commented:
Thanks to you, @rradczewski !! I changed *coderetreat* by *GDCR*, because I think we only register (and find) events for the Global Day of Coderetreat (not all the coderetreats that are held along the year).

I agree with your idea! I've removed those entries (new version available) and I've added a card to talk about an alternative at the next meeting.

Thanks again, Raimo!
